x/evm/handler_test.go

```diff
@@ -525,3 +528,78 @@ func (suite *EvmTestSuite) TestErrorWhenDeployContract() {
 	suite.Require().Nil(err)
 	suite.Require().Equal(snapshotCommitStateDBJson, currentCommitStateDBJson)
 }
+
+func (suite *EvmTestSuite) TestRevertErrorWhenCallContract() {
```

Comment:
I think we should add this to the suite of contracts that serve as integration tests